docs/sphinx/reference-properties.rst

```diff
@@ -664,6 +664,8 @@ Property Modification Functions
    :param    val:  The actual string value stored and will be returned by :c:func:`obs_data_get_string`
    :returns: The index of the list item.
 
+   `N.B.` : If ``p`` is of type OBS_COMBO_TYPE_EDITABLE, :c:func:`obs_data_get_string` will return ``name`` instead of ``val``.
```

Comment:
Please use "Note: " instead of "N.B.: ". If "Note: " doesn't fit well, then just omit it entirely.

See here and throughout.